### ISSUE TITLE
High Memory Consumption Issue

### ISSUE THREAD

**Issue by imtadashi** [sipsorcery-org/sipsorcery#1245]
Hi Sipsorcery team, I have been using Sipsorcery for few weeks and found memory consumption increases gradually. Is this normal? Even after stopping a PeerConnection, this memory is not disposed. Is there a way to forcefully dispose memory consumed by a PeerConnection or VideoStream?

**Comment by sipsorcery** [comment]
What sort of memory consumption is involved? Are you using any video encoding?

**Comment by ha-ves** [comment]
I can't replicate this with `SIPSorceryMedia.FFmpeg` (Sending & Receiving).

RTP framing and depacketizer each do copies in their process but they look to be garbage collected correctly.

**Comment by sipsorcery** [comment]
> Few minutes after the PeerConnection was closed, memory is cleared. But is it normal to have memory consumption while receiving RTPPackets?

Yes, particularly with video streams. There are a lot of buffer copies required which all add up.

**Comment by sipsorcery** [comment]
PR's welcome.
